### ISSUE TITLE
NTR, pulmo: Anthracosis

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6383]
**Preferred term label:** 
Anthracosis of the lung

**Synonyms** 
black discoloration of the lung, accumulation of carbon in the lungs

**Definition (free text, please give PubMed ID)**
black discoloration of bronchial mucosa, due to repeated exposure to air Pollution or Inhalation of smoke or coal dust particles. 
PMID: 25852756

Child-Terms: 
anthracotic lymphadenopathy: scattered foci or black spots, which retract mucosa inward
bronchial anthracofibrosis: occlussion of bronchial lumen, due to anthracosis of the lungs

**Parent term (use hpo.jax.org/app)**
Abnormal lung morphology HP:0002088

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
tuberculosis (TB) ORPHA:3389
bronchial anthracofibrosis (BAF) (no ORPHA or OMIM ID)

**Your nano-attribution (ORCID)**
LMU:chrapp, @katrinknoflach, @mgriese

**Comment by pnrobinson** [comment]
@chrapp, @katrinknoflach, @mgriese 
This seems to be a bronchial finding, e.g., see Figure 1 here
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4386010/
Would it be better to have this be the parent term?
https://hpo.jax.org/app/browse/term/HP:0025426

**Comment by mgriese** [comment]
Anthracosis can be in the airways and the alveoli, as well as all tissue components which are drained from there.
I would prefer this definition:
def: 
Anthracosis is the deposition of black carbon pigment. It is commonly seen in perihilar lymph nodes as well as within lung parenchyma. There can be prominent storiforming of histiocytes, to such an extent as to mimic a neoplastic lesion
histologically. It is a common finding in smokers and polluted city dwellers.
PMCID: PMC7262960

Abnormal lung morph is appropriate parent term

**Comment by pnrobinson** [comment]
Adding the term as suggested.
I added an explanation of storiforming, which is as word I was not familiar with:
Anthracosis is the deposition of black carbon pigment. It is commonly seen in perihilar lymph nodes as well as within lung parenchyma. There can be prominent storiforming (i.e., spiral patterning) of histiocytes, to such an extent as to mimic a neoplastic lesion histologically. Anthracosis is a common finding in smokers and polluted city dwellers.

**Comment by pnrobinson** [comment]
added term, thanks!
